cli/pull.go

```diff
@@ -86,6 +87,7 @@ func NewPullCmd(rootOpts *RootOptions) *cobra.Command {
 	cmd.Flags().StringVarP(&o.Output, "output", "o", o.Output, "output location for artifacts")
 	cmd.Flags().StringVar(&o.AttributeQuery, "attributes", o.AttributeQuery, "attribute query config path")
 	cmd.Flags().BoolVar(&o.PullAll, "pull-all", o.PullAll, "pull all linked collections")
+	cmd.Flags().BoolVarP(&o.NoVerify, "no-verify", "", o.NoVerify, "Skip Collection signature verification")
```

Comment:
```suggestion
	cmd.Flags().BoolVarP(&o.NoVerify, "no-verify", "", o.NoVerify, "skip collection signature verification")
```
Just to be consistent